api/services/v1alpha1/gateway_types.go

```diff
@@ -79,6 +79,19 @@ type GatewayConfigSpec struct {
 	// NetworkPolicy configuration for kube-auth-proxy
 	// +optional
 	NetworkPolicy *NetworkPolicyConfig `json:"networkPolicy,omitempty"`
+
+	// ProviderCASecretName is the name of the secret containing the CA certificate for the authentication provider
+	// Used when the OAuth/OIDC provider uses a self-signed or custom CA certificate.
+	// Secret must exist in the openshift-ingress namespace and contain a 'ca.crt' key with the PEM-encoded CA certificate.
+	// +optional
+	ProviderCASecretName string `json:"providerCASecretName,omitempty"`
+
+	// InsecureSkipVerify disables TLS certificate verification when connecting to the authentication provider.
+	// WARNING: This should ONLY be used in development/testing environments. Setting this to true
+	// disables all certificate validation and makes connections vulnerable to man-in-the-middle attacks.
+	// For production use with certificates, use ProviderCASecretName instead.
+	// +optional
+	InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty"`
```

Comment:
should we use postivie than negative name?
e.g verifyTLS=[true|false] default to true if omit